content/zh/docs/tasks/administer-cluster/manage-resources/cpu-constraint-namespace.md

```diff
@@ -13,25 +15,36 @@ weight: 40
 <!-- overview -->
 
 <!--
-This page shows how to set minimum and maximum values for the CPU resources used by Containers
-and Pods in a namespace. You specify minimum and maximum CPU values in a
-[LimitRange](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#limitrange-v1-core)
+This page shows how to set minimum and maximum values for the CPU resources used by containers
+and Pods in a {{< glossary_tooltip text="namespace" term_id="namespace" >}}. You specify minimum
+and maximum CPU values in a
+[LimitRange](/docs/reference/kubernetes-api/policy-resources/limit-range-v1/)
 object. If a Pod does not meet the constraints imposed by the LimitRange, it cannot be created
 in the namespace.
 -->
-本页介绍如何为命名空间中容器和 Pod 使用的 CPU 资源设置最小和最大值。
+ 本页介绍如何设置{{< glossary_tooltip text="命名空间" term_id="namespace" >}}中的容器和 Pod 使用的 CPU 资源的最小和最大值。
 你可以通过
-[LimitRange](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#limitrange-v1-core)
-对象声明 CPU 的最小和最大值. 如果 Pod 不能满足 LimitRange 的限制，它就不能在命名空间中创建。
+[LimitRange](/zh/docs/reference/kubernetes-api/policy-resources/limit-range-v1/)
```

Comment:
This article is not translated yet.

<https://kubernetes.io/zh/docs/reference/kubernetes-api/policy-resources/limit-range-v1/>

Reply:
So let's use the English version of the address first, and then modify the address here after `limit-range-v1`  is translated into Chinese?